### DISCUSSION TITLE
read-only mode

### DISCUSSION THREAD

**Discussion by bmillemathias** [outline/outline#6424] [General] [answered]
Hello,

I need to move the hosting of outline from docker to full k8s, and I have to deny people modifying the documents (hence the database modification) while I perform the migration. I did not see such options in outline administration or in the env file, is there a way to do this ?

thanks

**Comment by tommoor** [comment, marked as answer]
I'm afraid there isn't a way to do this right now, denying all modifications at a global level would be quite difficult – even opening a document performs many database writes

**Reply by bmillemathias** [reply]
Thanks

**Comment by includesec-erik** [comment]
@bmillemathias this is a DevOps topic about system migration. I'm sure a lot of other users are/have been/will be in a similar situation as yourself. 

In general you're likely to have some down time for the migration, even if it's momentarily. If you do create some scripts that help you with the migration, please do share with the community. And not to speak for the maintainers, but if they're good...then perhaps they might even be incorporated into the project if people find them useful!

**Reply by bmillemathias** [reply]
Hi

The audience is quite limited so I can sustain to have a downtime for 1h,  I just wanted to reduce it, so that's unlikely I'll invest time to develop script.

Thanks and have a nice day
